Cargo.toml

```diff
@@ -38,3 +38,4 @@ rmp-serde = "^0.13.7"
 benchmarks = []
 std = []
 default = ["serde", "std"]
```

Comment:
How about adding use-pkg-config to defaults?

Reply:
That would be my preferred solution, this would be a breaking change though. Do we want to do that regardless? Do we still want to allow control with an environment variable like `SODIUM_USE_PKG_CONFIG`?

Reply:
I don't think it is something we need in default features at the moment. It is fine as is.

Reply:
Ah, I see. If feature `use-pkg-config` isset we should use pkg config, otherwise fallback to the previous solution with SODIUM_USE_PKG_CONFIG. Resolved